articles/how-to-compile-bitcoin-core-and-run-the-tests.html

```diff
@@ -93,8 +93,7 @@
                       installed):
                       <code>
                         brew install automake berkeley-db4 libtool boost
-                        miniupnpc openssl pkg-config protobuf python qt libevent
-                        qrencode ccache
+                        miniupnpc pkg-config python qt libevent qrencode
```

Comment:
Thanks! I added `ccache` as it's basically essential to speeding up multiple builds. That said, it's not needed if you build only once. What do you think? Keep `ccache` in both the macOS and Linux dependencies, or remove it from both?

Reply:
Oh definitely keep. Yeah I didn't give any thought/investigation to why some had been removed. I think ccache should be in the Core doc too.

Reply:
Ok, can you update and then I'll merge.